tests/libdv/dv.go

```diff
@@ -133,3 +148,60 @@ func WithForceBindAnnotation() dvOption {
 func randName() string {
 	return "test-datavolume-" + rand.String(dvRandomNameLength)
 }
+
+// PVC Options
```

Comment:
I think this is redundant because we have the return argument.

Reply:
I think it's more clear that way, so new dv/pvc options will be placed where their friends are.